### PR TITLE
Gen3 Monthly Release 2023.04 qa-heal.planx-pla.net 1680794914

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2023.04 to qa-heal.planx-pla.net

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)
- peregrine
  - The `pidgin` service can be removed. Its functionalities are available at peregrine's `/coremetadata` endpoint (https://github.com/uc-cdis/peregrine/pull/198)
  - ASK AN ADMINISTRATOR TO DELETE THE PIDGIN DEPLOYMENT if you remove pidgin. This is NOT automatic (https://github.com/uc-cdis/peregrine/pull/198)
  - Peregrine does not rely on configuration files in the cloud-automation repository anymore. The Peregrine configuration files are in the Peregrine repository. (https://github.com/uc-cdis/peregrine/pull/191)
- sheepdog
  - Sheepdog does not rely on configuration files in the cloud-automation repository anymore. The Sheepdog configuration files are in the Sheepdog repository. (https://github.com/uc-cdis/sheepdog/pull/387)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/338)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)
- peregrine
  - The `pidgin` service can be removed. Its functionalities are available at peregrine's `/coremetadata` endpoint (https://github.com/uc-cdis/peregrine/pull/198)
  - ASK AN ADMINISTRATOR TO DELETE THE PIDGIN DEPLOYMENT if you remove pidgin. This is NOT automatic (https://github.com/uc-cdis/peregrine/pull/198)
  - Peregrine does not rely on configuration files in the cloud-automation repository anymore. The Peregrine configuration files are in the Peregrine repository. (https://github.com/uc-cdis/peregrine/pull/191)
- sheepdog
  - Sheepdog does not rely on configuration files in the cloud-automation repository anymore. The Sheepdog configuration files are in the Sheepdog repository. (https://github.com/uc-cdis/sheepdog/pull/387)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/338)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)
- peregrine
  - The `pidgin` service can be removed if `data-portal` is on version 5.4.2/2023.05 or newer. Its functionalities are available at peregrine's `/coremetadata` endpoint (https://github.com/uc-cdis/peregrine/pull/198)
  - ASK AN ADMINISTRATOR TO DELETE THE PIDGIN DEPLOYMENT if you remove pidgin. This is NOT automatic (https://github.com/uc-cdis/peregrine/pull/198)
  - Peregrine does not rely on configuration files in the cloud-automation repository anymore. The Peregrine configuration files are in the Peregrine repository. (https://github.com/uc-cdis/peregrine/pull/191)
- pidgin
  - If you are seeing this message, you are deploying the Pidgin service after it has been deprecated. If you are using Peregrine version 3.2.0/2023.04 or newer, the Pidgin service can and should be removed. (https://github.com/uc-cdis/pidgin/pull/48)
- sheepdog
  - Sheepdog does not rely on configuration files in the cloud-automation repository anymore. The Sheepdog configuration files are in the Sheepdog repository. (https://github.com/uc-cdis/sheepdog/pull/387)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/338)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)
- peregrine
  - The `pidgin` service can be removed if `data-portal` is on version 5.4.2/2023.05 or newer. Its functionalities are available at peregrine's `/coremetadata` endpoint (https://github.com/uc-cdis/peregrine/pull/198)
  - ASK AN ADMINISTRATOR TO DELETE THE PIDGIN DEPLOYMENT if you remove pidgin. This is NOT automatic (https://github.com/uc-cdis/peregrine/pull/198)
  - Peregrine does not rely on configuration files in the cloud-automation repository anymore. The Peregrine configuration files are in the Peregrine repository. (https://github.com/uc-cdis/peregrine/pull/191)
- pidgin
  - If you are seeing this message, you are deploying the Pidgin service after it has been deprecated. If you are using Peregrine version 3.2.0/2023.04 or newer AND data-portal version 5.4.2/2023.05 or newer, the Pidgin service can and should be removed. (https://github.com/uc-cdis/pidgin/pull/48)
- sheepdog
  - Sheepdog does not rely on configuration files in the cloud-automation repository anymore. The Sheepdog configuration files are in the Sheepdog repository. (https://github.com/uc-cdis/sheepdog/pull/387)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/338)

**Review by mfshao — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)
- peregrine
  - The `pidgin` service can be removed if `data-portal` is on version 5.4.2/2023.05 or newer. Its functionalities are available at peregrine's `/coremetadata` endpoint (https://github.com/uc-cdis/peregrine/pull/198)
  - ASK AN ADMINISTRATOR TO DELETE THE PIDGIN DEPLOYMENT if you remove pidgin. This is NOT automatic (https://github.com/uc-cdis/peregrine/pull/198)
  - Peregrine does not rely on configuration files in the cloud-automation repository anymore. The Peregrine configuration files are in the Peregrine repository. (https://github.com/uc-cdis/peregrine/pull/191)
- pidgin
  - If you are seeing this message, you are deploying the Pidgin service after it has been deprecated. If you are using Peregrine version 3.2.0/2023.04 or newer AND data-portal version 5.4.2/2023.05 or newer, the Pidgin service can and should be removed. (https://github.com/uc-cdis/pidgin/pull/48)
- sheepdog
  - Sheepdog does not rely on configuration files in the cloud-automation repository anymore. The Sheepdog configuration files are in the Sheepdog repository. (https://github.com/uc-cdis/sheepdog/pull/387)
## Breaking changes
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (https://github.com/uc-cdis/indexd/pull/338)

File: qa-heal.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,28 +8,28 @@
...
-    "requestor": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/requestor:2023.02",
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2023.02",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2023.02",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2023.02",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2023.02",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2023.02"
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2023.04",
+    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2023.04",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2023.04",
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2023.04",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:25]: I think we can remove this service now?

Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json:25]: we need to change portal version in that case `AND data-portal version 5.4.2/2023.05`

**Pull request merged**